src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -60,13 +60,13 @@ private static void InitColorTable()
             // SystemColors. In order to avoid a static dependency on SystemEvents since it is not available on all platforms 
             // and as such we don't want to bring it into the shared framework. We use the desktop identity for SystemEvents
             // since it is stable and will remain stable for compatibility whereas the core identity could change.
-            Type systemEventsType = Type.GetType("Microsoft.Win32.SystemEvents, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", throwOnError: false);
+            Type systemEventsType = Type.GetType("Microsoft.Win32.SystemEvents, Microsoft.Win32.SystemEvents", throwOnError: false);
```

Comment:
Just a question:- is there any particular reason for changing the System to Microsoft.Win32.SystemEvents ?

Reply:
Stephen and Jan discussed it here: https://github.com/dotnet/corefx/pull/39315#discussion_r301565480. With a bit of luck, it avoids loading the System.dl facade and saves a lot of time. If we don't have the luck, it still avoids resolution of the type forward. Fragility is not a concern per the discussion.